geode-membership/build.gradle

```diff
@@ -38,9 +38,7 @@ dependencies {
     //Jgroups is a core component of our membership system.
     implementation('org.jgroups:jgroups')
 
-    testImplementation(project(':geode-junit')) {
-        exclude module: 'geode-logging'
-    }
+    testImplementation(project(':geode-junit'))
```

Comment:
each of these excludes was superfluous. the excludes were necessary in other modules, where the exclusions broke what would have been circular inter-module dependencies